post/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.vscode
```

Comment:
this folder is still in repository, remove and make commit without . vscode folder